website/docs/diff.md

```diff
@@ -24,11 +24,9 @@ slug: /diff/
 8. Database and collection names restrictions:
    * name cannot start with the reserved prefix `_ferretdb_`;
    * database name must not include non-latin letters, spaces, dots, dollars;
-   * collection name must not include non-latin letters, spaces, dots or dollars;
-   * name must not start with a number;
+   * collection name must be valid UTF-8 characters;
```

Comment:
Is it different from how MongoDB behaves, don't they need a valid UTF-8 string as well?

Reply:
Yes this is different
mongo
```
test> db.createCollection("Ä°nanÃ§")
{ ok: 1 }
```

FerretDB
```
test> db.createCollection("Ä°nanÃ§")
MongoServerError: Invalid collection name: 'test.Ä°nanÃ§'
```

Reply:
Oh, nice! Is there a diff test in `dance` for it?

Reply:
No, I'm doubting if this is worth putting it to dance.

Reply:
True, let me add it. :sweat_smile:

Reply:
That will be fixed :hugs: